site/docs/providers/aws-bedrock.md

```diff
@@ -188,6 +188,42 @@ Nova models use a slightly different configuration structure compared to other B
 
 :::
 
+### Amazon Nova Sonic Model
+
+The Amazon Nova Sonic model (`amazon.nova-sonic-v1:0`) is a multimodal model that supports audio input and text/audio output with tool-using capabilities. It has a different configuration structure compared to other Nova models:
+
+```yaml
+providers:
+  - id: bedrock:amazon.nova-sonic-v1:0
+    config:
+      inferenceConfiguration:
+        maxTokens: 1024 # Maximum number of tokens to generate
+        temperature: 0.7 # Controls randomness (0.0 to 1.0)
+        topP: 0.95 # Nucleus sampling parameter
+      textOutputConfiguration:
+        mediaType: text/plain
+      toolConfiguration: # Optional tool configuration
+        tools:
+          - toolSpec:
+              name: 'getDateTool'
+              description: 'Get information about the current date'
+              inputSchema:
+                json: '{"$schema":"http://json-schema.org/draft-07/schema#","type":"object","properties":{},"required":[]}'
```

Comment:
Is `inputSchema` required in this example since (assumedly) the date getter would have no args?

Reply:
I believe inputschema is required, even when there are no args- https://docs.aws.amazon.com/nova/latest/userguide/speech-tools-use.html (See the example with the provided schema with no args). 

That said, the error given with a malformed tool config is very opaque so I'd recommend following the example where possible.